dockerfile/cargo/Dockerfile

```diff
@@ -82,18 +82,19 @@ RUN set -eux;\
     if [ ! -z "$GO_VERSION" ]; then export PATH=$PATH:/usr/local/go/bin; fi;\
     if [ "$TARGETARCH" = "arm64" ]; then export ARCH=aarch64 CAPS=AARCH64;\
     elif [ "$TARGETARCH" = "amd64" ]; then export ARCH=x86_64 CAPS=x86_64; fi;\
+    export CARGO_BUILD_TARGET=${ARCH}-unknown-linux-gnu;\
     if [ "$TARGETARCH" != "$BUILDARCH" ]; then\
       export CARGO_TARGET_${CAPS}_UNKNOWN_LINUX_GNU_LINKER=${ARCH}-linux-gnu-gcc\
         CC_${ARCH}_unknown_linux_gnu=${ARCH}-linux-gnu-gcc\
         CXX_${ARCH}_unknown_linux_gnu=${ARCH}-linux-gnu-g++\
         PKG_CONFIG_SYSROOT_DIR=/usr/${ARCH}-linux-gnu;\
     fi;\
     [ ! -z "$PRE_BUILD" ] && sh -c "${PRE_BUILD}";\
-    [ ! -z "$BUILD_ENV" ] && export ${BUILD_ENV};\
-    [ ! -z "$BUILD_TAGS" ] && export "${BUILD_TAGS}";\
-    if [ ! -z "$BUILD_DIR" ]; then cd "${BUILD_DIR}"; fi;\
     if [ ! -z "$BUILD_TARGET" ]; then\
-        cargo ${BUILD_TARGET} --target ${ARCH}-unknown-linux-gnu;\
+      if [ ! -z "$BUILD_ENV" ]; then export ${BUILD_ENV}; fi;\
```

Comment:
Is it okay that `${BUILD_ENV}` is not quoted here, even though the next line quotes `${BUILD_TAGS}`?

Reply:
yes, curly brackets are unnecessary unless there is a `([a-z][A-Z]_)` character immediately following the var name which should not be included in the var name. Consistency is good though.

Reply:
It's not necessary to produce valid Bash syntax, but it has subtle differences when the variable contains spaces. See https://www.shellcheck.net/wiki/SC2086. I was wondering if this was a deliberate unquoting to force whitespace treatment as separate shell arguments.